src/utils/tooltip.js

```diff
@@ -109,34 +111,6 @@ const NodeTooltip = (props) => {
     );
 };
 
-const NodeEndpointsTooltip = (props) => {
```

Comment:
I don’t like two things in these tooltips:
1. redux reducer as additional level - it complicates code
2. two different ways to display tooltips in the app - this redux tooltip and different components from uikit

So I prefer to use Popover, Popup and Tooltip from uikit, it makes it easier and more consistent. Therefore I move this components from here to /components/Tooltips

Reply:
I totally agree with you! Redux Tooltip is awful! Let's get rid of it step by step.